.github/workflows/avm.platform.assign-issue.yml

```diff
@@ -0,0 +1,38 @@
+# Workflow for notifying and assigning issues on creation
+name: Issue Creation - notify and assign team
+
+on:
+  issues:
+    types: [opened]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch: {}
+
+jobs:
+  NotifyTeams:
+    name: Notify-Team
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: "Checkout"
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - env:
+          GH_TOKEN: ${{ github.token }}
+        name: "Run scripts"
+        shell: pwsh
+        run: |
+          # Load used functions
+          . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines'  'platform' 'Assign-Issue.ps1')
+
+          $functionInput = @{
+            Repo = "${{ github.repository_owner }}/${{ github.event.repository.name }}"
+            IssueUrl = "${{ github.event.issue.url }}"
+          }
+
+          Write-Verbose "Invoke task with" -Verbose
+          Write-Verbose ($functionInput | ConvertTo-Json | Out-String) -Verbose
+
+          Set-Issue @functionInput -Verbose # -WhatIf
```

Comment:
`Set-Issue` is very opened ended and thinking about future scale we problem want to have descriptive names for our functions

Reply:
To be fair, @rahalan is not just adding an owner (which would justify naming it `Set-GitHubIssueOwner` but also adds a comment and  adds labels.

Would you have a suggestion?

Reply:
Maybe `Set-GitHubIssueOwnerConfig` ?

Reply:
Set-AvmGitHubIssueOwnerConfig

Reply:
Done